common.js

```diff
@@ -0,0 +1,142 @@
+/* eslint no-console:0 */
+//
+// Use a Custom Resource Definition to extend the Kubernetes API and the client.
+//
+var exports = module.exports = {};
+let fs = require('fs');
+const path = require('path');
+
+exports.delete_env = async function (client, env_name, nmspace) {
+
+    const del_env = await client.apis['fission.io'].v1.namespaces(nmspace).environments(env_name).delete();
+    console.log("Fission envirnment ",del_env['body']['details']['name']," deleted");
+}
+exports.delete_fn = async function (client, fn_name, nmspace) {
+    const del_fn = await client.apis['fission.io'].v1.namespaces(nmspace).functions(fn_name).delete();
+    console.log("Fission function ",del_fn['body']['details']['name']," deleted");
+}
+exports.fn_code = async function (client, fn_name, nmspace) {
+
+    const fn_data = await client.apis['fission.io'].v1.namespaces(nmspace).functions(fn_name).get();
+    const pkg_name = fn_data['body']['spec']['package']['packageref']['name'];
+    const pkg_namespace = fn_data['body']['spec']['package']['packageref']['namespace'];
+    const pkg_data = await client.apis['fission.io'].v1.namespaces(pkg_namespace).packages(pkg_name).get();
+    const pkg_code = pkg_data['body']['spec']['deployment']['literal'];
+    const pkg_decode = new Buffer(pkg_code, 'base64');
+    console.log(pkg_decode.toString());
```

Comment:
what's the use of this console.log, either print some message along with variable values.

Reply:
This shows the source code of the function that you deployed. Fission gives a similar output as mine hence I am using the same

Reply:
Ok, but atleast print some message before printing the source code, so that whenever anyone check logs he will be able to understand what it is.

Reply:
ok will do